test/e2e/specs/editor/various/block-bindings.spec.js

```diff
@@ -47,7 +47,7 @@ test.describe( 'Block bindings', () => {
 		} );
 
 		test.describe( 'Paragraph', () => {
-			test( 'Should show the value of the custom field', async ( {
+			test( 'should show the value of the custom field', async ( {
```

Comment:
Why enlarge the diff with these changes?

Reply:
I realized while working on this pull request that seemed to be the standard so I took the opportunity to change that. Although you're right that I should have probably created a different PR. Sorry for that.